Add email guidelines to handbook (#1106)
flowforge/website: src/handbook/marketing/email.md

---

> meta:

Author: Sure, how do I do that?

> meta:

Reviewer: Add a line to `src/handbook/marketing/index.md`

> meta:

Author: done

## Types of Email

### Transactional email with prospective customers and current customers

The sales team, customer success and support team will be required to send email to communicate with specific potential customers and customers. These types of email are typically between the FlowFuse employee and an individual customer to accomplish a task or transaction. For instance, handling a support request or further a sales process.

These types of email should be stored in HubSpot using the HubSpot interface or the HubSpot Chrome plugin. The goal should be that all transactional email is associated with the contact in Hubspot to have an history of the interaction.

### Community email 

The marketing team is responsible for sending email to all community members, potential customers and customers that provide updates on FlowFuse and events organized by FlowFuse. Examples of these email include monthly newsletter, webinar promotion, product release, major company announcements, etc.

> These types of emails should be sent via HubSpot. 

Author: The marketing team is responsible so only the marketing team should be sending these emails.  People can certainly request an email to be sent but it needs to be sent by the marketing team.

> These types of emails should be sent via HubSpot. 

Reviewer: Can you update the handbook to say so?

> These types of emails should be sent via HubSpot. 

Author: Done

### FlowFuse Cloud Trial email

The marketing team, in collaboration with the customer success team, is responsible for defining and creating an email campaign to encourage the successful trial of FlowFuse Cloud. 

These emails will be sent from HubSpot using data based on the user interaction with FlowFuse Cloud. A system has been put in place to query the FF Cloud database to create events that are added to HubSpot. HubSpot workflows will trigger the emails based on these events.

The expectation is that the marketing team will continuously test different types of email, and email content to improve the conversion rate of the trial period.

> 

Reviewer: Who does anyone propose changes to these emails?

Author: Anyone can open a issue in github to request a change. I can add this to the document if you want.

Author: Done

The following guidelines are meant to document the roles and responsibilities of sending emails to ensure we have a consistent and well defined approach.

## Types of Email

### Transactional email with prospective customers and current customers

The sales team, customer success, and support team will be required to send email to communicate with specific potential customers and customers. These types of email are typically between the FlowFuse employee and an individual customer to accomplish a task or transaction. For instance, handling a support request or further a sales process.

These types of email should be stored in HubSpot using the HubSpot interface or the HubSpot Chrome plugin. The goal should be that all transactional email is associated with the contact in Hubspot to have an history of the interaction.

### Community email 

The marketing team is responsible for sending email to all community members, potential customers and customers that provide updates on FlowFuse and events organized by FlowFuse. Examples of these email include monthly newsletter, webinar promotion, product release, major company announcements, etc.

> These types of emails should be sent via HubSpot. To ensure consistency and continuity, the marketing team should be the only team sending community emails via HubSpot. Anyone can request an email to be sent by opening a github issue on the website.

Reviewer: ```suggestion
These types of emails should be sent via HubSpot. To ensure consistency and continuity, the marketing team should be the only team sending community emails via HubSpot. Anyone can request an email to be sent by opening a [GitHub issue on the website repository](https://github.com/flowforge/website/issues).
```

> ### FlowFuse Cloud Trial email

Reviewer: ```suggestion
### FlowFuse Cloud User emails
```

> The marketing team, in collaboration with the customer success team, is responsible for defining and creating an email campaign to encourage the successful trial of FlowFuse Cloud. 

Reviewer: ```suggestion
The marketing team, in collaboration with the customer success team, is responsible for defining and creating an email campaign to encourage the successful usage of FlowFuse Cloud and users growth in adoption.
```

Author: how about we say 'successful usage and adoption of FlowFuse Cloud.'

These emails will be sent from HubSpot using data based on the user interaction with FlowFuse Cloud. A system has been put in place to query the FF Cloud database to create events that are added to HubSpot. HubSpot workflows will trigger the emails based on these events.

> The expectation is that the marketing team will continuously test different types of email, and email content to improve the conversion rate of the trial period.

Reviewer: ```suggestion
The expectation is that the marketing and customer success team will continuously test different types of email, and email content to improve the conversion rate of the trial period and adoption beyond of FlowFuse Cloud.
```

Author: Not sure what you mean by 'and adopt beyond of'?

Reviewer: For example; growing from starter package to the Team tier. But also other utilisation topics, like growing starter users to adopt devops pipelines.

> FlowFuse Cloud Trial emails should only be sent by the marketing team via HubSpot. Anyone can suggestion changes by opening a github issue. 

Reviewer: ```suggestion
FlowFuse Cloud Trial emails should only be sent by the marketing team via HubSpot. Requesting changes to these emails is done by opening a
[GitHub issue on the website repository](https://github.com/flowforge/website/issues).
```
